src/wings_palette.erl

```diff
@@ -98,6 +98,11 @@ forward_event({apply, ReturnSt, Fun}, Window, St0) ->
 	false ->
 	    Fun(St0)
     end;
+forward_event({new_state, #st{}=St}=_, _Window, _) ->
+    %% wings_dialog functions sends {new_state, St} to this window,
+    %% so send it to geom
+    wings_wm:send(geom, {new_state,St}),
```

Comment:
wings_wm sends new_state to all windows that subscribes to them, it should just be ignored by this window,
sending it geom will work but cause every update to be handled twice, if I'm not mistaken.
Or it should be handled as current_state at line 81.

But anyway we should not send it to the Window process, which will cause send a copy of the model 
to the palette window, which will increase the memory usage many times,
since the undo history will be expanded.